keyboards/bbrfkr/dynamis/dynamis.h

```diff
@@ -23,7 +23,8 @@
       C10, C11, C12, C13, C14, C15, C16, C66, C65, C64, C63, C62, C61, C60, \
       C20, C21, C22, C23, C24, C25, C26, C76, C75, C74, C73, C72,      C70, \
       C30, C31, C32, C33, C34, C35, C36, C86, C85, C84, C83, C82, C81, C80,      C91, \
-      C40, C41, C42,      C43,      C44,           C45, C46, C96, C95,      C94, C93, C92 \
+      C40, C41, C42,      C43,      C44,           C45, C46, C96, C95,      C94, C93, C92, \
+      C100, C101 \
```

Comment:
These are sole for encoder support? 

If so, it needs to be removed.  VIA implementation should not be leaked into keyboard level code.

Reply:
Ok. So, How the behaviors of encoder for each layers should be implemented?

Reply:
Either per the normal callback (encoder_update_kb) in the keyboard's C file.  This provides a default/fallback behavior if there is not a user function, and allows the user function to override the behavior if it is present. 

Alternatively, you can enable the encoder map feature at the keymap level.

Reply:
I see. I updated to control with whether `layer_state == 0` or not.